capstone/src/main/java/com/google/univiz/SuggestionDataApi.java

```diff
@@ -0,0 +1,15 @@
+package com.google.univiz;
+
+import java.util.List;
+
+/**
+ * Communicates directly with the REST API to get a list of college suggestions related to a client
+ * query
+ */
+public interface SuggestionDataApi {
+  /**
+   * Takes a list of college names and returns a list of SuggestionData objects, where each object
```

Comment:
Can you true up this comment to match the method?
* No longer a list argument.
* Also s/providede/provided/.

Reply:
updated!